### PR TITLE
✨ Move concrete product api bundle from fond-of-spryker to fond-of-oryx

### PR DESCRIPTION
**Pull request by DerCalli**
- use new namespace (FondOfOryx)
- add ci pipeline for split repo
- remove unused query container
- add required transfer definition
- add phpstan config
- exclude orm and transfer classes from code coverage
- add better unit testing
- ...

### CODE REVIEW

File: bundles/concrete-product-api/LICENSE [hunk 1]
```diff
@@ -0,0 +1,9 @@
+The MIT License (MIT)
+
+Copyright (c) 2021 FOND OF GmbH
```
Comment (sourcecube) [inline bundles/concrete-product-api/LICENSE:3]: Update copyright to 2022

File: bundles/concrete-product-api/composer.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  "require": {
+    "php": ">=7.3",
+    "spryker/api": "^0.1.0 || ^0.2.0 || ^0.3.0",
+    "spryker/api-query-builder": "^0.1.0",
+    "spryker/kernel": "^3.30.0",
+    "spryker/product": "^5.0.0 || ^6.0.0",
+    "spryker/propel-orm": "^1.2.0",
+    "spryker/symfony": "^3.0.0",
+    "spryker/util-date-time": "^1.0.0"
+  },
```
Comment (sourcecube) [inline bundles/concrete-product-api/composer.json:24]: Remove unnecessary packages

**Review by sourcecube — APPROVED** [review]

File: bundles/concrete-product-api/LICENSE (hunk 1 above)
Comment (julianzimmermann) [inline bundles/concrete-product-api/LICENSE:3]: 2022

File: bundles/concrete-product-api/composer.json [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  "require-dev": {
+    "fond-of-codeception/spryker": "^1.0",
+    "spryker/code-sniffer": "*",
+    "spryker-sdk/phpstan-spryker": "*",
+    "roave/security-advisories": "dev-latest"
```
Comment (julianzimmermann) [inline bundles/concrete-product-api/composer.json:29]: phpstan missing

**Review by julianzimmermann — APPROVED** [review]
also github action workflow is missing

**Review by SpiderDan98 — APPROVED** [review]

**Review by julianzimmermann — APPROVED** [review]

**Review by gengjozsef — APPROVED** [review]

**Pull request merged**